build.py

```diff
@@ -410,6 +410,14 @@ def do_make(target=None, log=False):
             print "ERROR: TREE_NAME not set, aborting publish step"
             publish = False
 
+    vmlinux_file = os.path.join(install_path, 'vmlinux')
+    if os.path.exists(vmlinux_file):
+        print "found vmlinux, parsing sizes"
+        import elf
+        bmeta.update(elf.read(vmlinux_file))
+        bmeta["vmlinux_file_size"] = os.stat(vmlinux_file).st_size
+        os.remove(vmlinux_file)
```

Comment:
Why remove the vmlinux file locally.  

This will cause make to rebuild on subsequent rebuilds, even if source hasn't changed.